hotline/users/views.py

```diff
@@ -107,23 +107,18 @@ def home(request):
     })
 
 
-@permissions.can_view_user
-def detail(request, user_id):
-    """
-    A nice homepage for user the user. By passing the user_id as a parameter,
-    this allows admins to view the user's detail page, without having to
-    masquerade as them.
-    """
-    user = get_object_or_404(User, pk=user_id)
+class detail(DetailView):
```

Comment:
You should rename the class to match Python conventions. Originally this was a function based view, which is why the name was lowercased.